### PR TITLE
Progress bar styling

### PR DESCRIPTION
**Pull request by abilpraju-aot**
https://aottech.atlassian.net/browse/FWF-1795
image
image

### CODE REVIEW

File: src/stories/Introduction.stories.mdx [hunk 1]
```diff
@@ -226,8 +226,69 @@ number of tabs and much more limitations of content within the headings. Conside
 [TODO]
 
 ## Progress Indicators
-[TODO]
+progress bar indicates that the user's request has been received and the application is making progress toward completing the requested action.
+
+<div class="formsflow-progressbar-container formsflow-progressbar-success" style={{height:"2px"}}>
+    <div class="formsflow-progressbar-count" aria-valuenow="90%" style={{"left":"90%"}}></div>
+     <div class="formsflow-progressbar">
+       <div class="" style={{"width":"90%"}}></div>
```
Comment (sreehari-aot) [inline src/stories/Introduction.stories.mdx]: remove empty class

File: src/stories/Introduction.stories.mdx [hunk 2]
```diff
@@ -226,8 +226,69 @@ number of tabs and much more limitations of content within the headings. Conside
...
+    </div>
+    <span>Success Message</span>
+</div>
+
+
+<div className="subheading">Usage</div>
+  
+
+  ```html dark
+    <div class="formsflow-progressbar-container formsflow-progressbar-success" style={{height:"2px"}}>
```
Comment (sreehari-aot) [inline src/stories/Introduction.stories.mdx]: use correct proper tabbing that might be the reason for parts of the code not getting highlighted in the screenshot

**Comment by sreehari-aot** [comment]
leave proper spacing between heading, component and code block

**Review by shuhaib-aot — CHANGES_REQUESTED** [review]
please add description for how to customize progress bar

**Comment by sreehari-aot** [comment]
please update the screenshots

**Review by shuhaib-aot — APPROVED** [review]

**Pull request merged**